client/verta/verta/registry/check_model_dependencies.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""Utility function for checking model dependencies against an environment
+to identify missing packages."""
+
+from typing import Set, Type
+import warnings
+
+from verta.registry import VertaModelBase
+from verta.environment import Python
+from verta._internal_utils import model_dependencies as md
+from verta._internal_utils._pip_requirements_utils import parse_req_spec
+
+
+def check_model_dependencies(
```

Comment:
Like I [mentioned last week](https://github.com/VertaAI/modeldb/pull/3641#discussion_r1125093066), we should hide this from our public API before merging since it's not yet in an end-user usable state.

Reply:
Made the function hidden.  Thanks for the reminder.